pinot-segment-local/src/main/java/org/apache/pinot/segment/local/utils/SchemaUtils.java

```diff
@@ -144,6 +144,10 @@ public static void validate(Schema schema, boolean isIgnoreCase) {
         if (fieldSpec.getFieldType().equals(FieldSpec.FieldType.DATE_TIME)) {
           validateDateTimeFieldSpec((DateTimeFieldSpec) fieldSpec);
         }
+        if (fieldSpec.getDataType().equals(FieldSpec.DataType.FLOAT)
+                || fieldSpec.getDataType().equals(FieldSpec.DataType.DOUBLE)) {
```

Comment:
(minor) Can you also reformat the changes in this file?

Reply:
Sorry :/